tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +521,361 @@ def cvg_api(snappi_api_serv_ip,
     yield api
     if getattr(api, 'assistant', None) is not None:
         api.assistant.Session.remove()
+
+
+def snappi_dut_base_config(duthost_list,
+                           tgen_ports,
+                           snappi_ports,
+                           snappi_api):
+    """
```

Comment:
Comments are inconsistent with definition of the function.

Reply:
this has been fixed